extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php

```diff
@@ -65,7 +65,8 @@ public static function onMakeGlobalVariablesScript( array &$aVars ) {
 
 		if ( $wgEnableVisualEditorExt && \VisualEditorHooks::isAvailable( \RequestContext::getMain()->getSkin() ) ) {
 			$aVars['wgEnablePortableInfoboxBuilderInVE'] = $wgEnablePortableInfoboxBuilderInVE &&
-				\RequestContext::getMain()->getUser()->isAllowed( 'createpage' );
+				// use dummy title in template ns
+				Title::newFromText( 'ATemplate', NS_TEMPLATE )->userCan( 'createpage' );
```

Comment:
What will happen if there is already the `Template:ATemplate` page? What if it's protected (`protected_titles` table)? Should we use a longer string that is very unlikely to be an actual title?